File: Data Structures & Algorithms/two-integer-sum-ii/submission-0.js
class Solution {
    /**
     * @param {number[]} numbers
     * @param {number} target
     * @return {number[]}
     */
    twoSum(numbers, target) {
        let maxIndex = numbers.length;

        for (let i = 0; i < numbers.length; i++) {
            const currentSum = numbers[i];
            for (let j = i+1; j < maxIndex; j++) {
                const sum = currentSum + numbers[j];

                if (sum > target) {
                    maxIndex = j
                }

                if (sum === target) {
                    return [i + 1, j + 1];
                }
            }
        }
    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-0.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        numSet = {}

        for i in range (len(nums)):
            difference = target - nums[i]
            if difference in numSet:
                return [numSet[difference], i]
            numSet[nums[i]] = i
                


